src/lib/exports.ts

```diff
@@ -313,13 +325,17 @@ function revokeAttachmentsUrls(attachments: string[]) {
   })
 }
 
+export function getValidNoteTitle(note: NoteDoc): string {
```

Comment:
note._id is not a title. Please return `Untitled`

Reply:
I want to return untitled, but if we do that, two notes within the same folder with no title would be named "Untitled.ext", which would overwrite the note contents.

So the solution is some kind of ID for untitled notes, either note._id or some index we could introduce when exporting notes.

Reply:
Hmm, how do you think appending note id after the title?

`Untitled-XXXXXXXX.md`
`Untitled-XXXXXXXX.md`

Reply:
That's great, if note id is something we can let users know of, I think that should be the easiest, safe solution.